src/components/Layout/LayoutHeader.tsx

```diff
@@ -112,14 +111,13 @@ export const LayoutHeader = ({
             <IconDoubleChevron />
             <VisuallyHidden>Close menu</VisuallyHidden>
           </Button>
-          {isGen2 ? null : (
-            <div className="layout-sidebar-platform">
-              <PlatformNavigator
-                currentPlatform={currentPlatform}
-                isPrev={isPrev}
-              />
-            </div>
-          )}
+
+          <div className="layout-sidebar-platform">
+            <PlatformNavigator
+              currentPlatform={currentPlatform}
+              isGen1={isGen1}
+            />
+          </div>
```

Comment:
Platform navigator should be shown all the time now. At the moment this component handles the version switcher too, so I updated it to accept whether or not we're on gen1